updated to reflect the new admin ui (#1525)
keycloak/keycloak-documentation: server_admin/topics/users/ref-user-credentials.adoc

You can manage credentials of a user in the *Credentials* tab. 

.Credential management

> image:{project_images}/user-credentials.png[user credentials]

Reviewer: Can you replace the screenshot with one that has two or more credentials?

Author: But this is the credential screen when you just created a user. I think it doesn't make sense to have something different in this case, the original screenshot also didn't have any credentials

ifeval::[{project_community}==true]

> When credentials have been created for this user a table with all the credentials will appear dragging and dropping the rows will shift the priority of the credential for the user.

Reviewer: ```suggestion
You change the priority of credentials by dragging and dropping rows. The new order determines the priority of the credentials for that user.
```

endif::[]
ifeval::[{project_product}==true]
This tab includes the following fields:

Position::
   The arrow buttons in the *Position* column allow you to shift the priority of the credential for the user.
endif::[]

>  The topmost credential has the highest priority. The priority determines which credential is displayed first after a user logs in.

Reviewer: ```suggestion
The topmost credential has the highest priority. The priority determines which credential is displayed first after a user logs in.
```

   credential.

Actions::
ifeval::[{project_community}==true]
   This column has two actions. Click *Reset passwrod* to change the passord for the user and *Delete* to remove the credential.

> endif::[]

Reviewer: This column is not in new version.  You can remove this.
There is a Vertical ellipsis with Delete, but I think they can figure that out.

ifeval::[{project_product}==true]
   This column has two actions. Click *Save* to record the value or the user field. Click *Delete* to remove the credential.
endif::[]

> You cannot configure other types of credentials for a specific user in the admin console; that task is the user's responsibility.

Reviewer: ```suggestion
You cannot configure other types of credentials for a specific user in the Admin Console; that task is the user's responsibility.
```
